pages/guideforms.py

```diff
@@ -916,6 +939,8 @@ def make_display_specs(*shared_field_names):
     'comments',
 ]
 
+DISPLAY_IN_ACCURACY_VERIFICATION = ['accurate_as_of']
```

Comment:
Why not include it on line 946 for DISPLAY_FIELDS_IN_STAGES['Metadata']?

Reply:
I was not sure if this would affect any of the forms that currently exist, as we don't want this field showing anywhere else. If it will not affect which fields display elsewhere, I will add it 👍